Source/Krypton Toolkit/Third Party/Microsoft.WindowsAPICodePack.Core/AppRestartRecovery/ApplicationRecoveryException.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Microsoft.WindowsAPICodePack.Core
```

Comment:
Why not pull this in as a nuget pack ?
Have you done specific fixes that are not included in V 1.1.1 -> 1.1.4?

![image](https://user-images.githubusercontent.com/2418812/124076041-a28de300-da3d-11eb-8067-8d34bcf40c37.png)

Reply:
No, just made the license agreement (MIT) mandatory, though it doesn't include the fixes contained in this repo: https://github.com/ljvankuiken/WindowsAPICodePack